frontend/server/src/Controllers/Course.php

```diff
@@ -3664,6 +3665,55 @@ private static function getCourseDetailsForLoggedUser(
         return $detailsResponse;
     }
 
+    /**
+     * Gets the course and specific assignment details
+     *
+     * @return array{smartyProperties: array{payload: AssignmentDetailsPayload, title: string, inContest: bool}, entrypoint: string}
+     */
+    private static function getAssignmentDetailsForTypescript(
```

Comment:
<3

sólo un pequeño nit:

```suggestion
    private static function getAssignmentDetailsForTypeScript(
```

el nombre del lenguaje es TypeScript, no Typescript. así es consistente con https://github.com/omegaup/omegaup/pull/5153 también.